### PR TITLE
[Bug](web) fix web of frontend meet error

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
# Proposed changes

java.lang.NoSuchMethodError: javax.servlet.http.HttpServletResponse.setContentLengthLong(J)V
https://github.com/apache/doris/pull/18882  remove some version of dependency
but the version of javax.servlet.api is required

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.63 seconds
 stream load tsv:          429 seconds loaded 74807831229 Bytes, about 166 MB/s
 stream load json:         21 seconds loaded 2358488459 Bytes, about 107 MB/s
 stream load orc:          60 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230504120610_clickbench_pr_138201.html

File: fe/fe-core/pom.xml [hunk 1]
```diff
@@ -172,6 +172,7 @@ under the License.
         <dependency>
             <groupId>javax.servlet</groupId>
             <artifactId>javax.servlet-api</artifactId>
+            <version>3.1.0</version>
```
Comment (CalvinKirs) [inline fe/fe-core/pom.xml:175]: Please define it in `dependency` manage in the parent pom, we need to manage dependencies uniformly.

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by morningman** [comment]
CC @CalvinKirs

**Review by CalvinKirs — APPROVED** [review]

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**